js/password/pwEvent.js

```diff
@@ -1,35 +1,39 @@
-import base from "./base.js";
-import errorMsg from "./errorMsg.js";
+import base from "../function/regex.js";
+import errorMsg from "../function/errorMsg.js";
 
 const passwordInput = document.getElementById('password');
 const passwordRepeatInput = document.getElementById('password-repeat');
-const userPassword = 'codeit101'
+const userPassword = 'sprint101'
 
 function validatePassword() {
   const passwordValue = passwordInput.value.trim();
   const url = window.location.href;
-  if (passwordValue === '') {
+  if (!passwordValue) {
     errorMsg.ErrorMessage(passwordInput, '비밀번호를 입력해 주세요.');
     errorMsg.boderLine('boder-error', true);
     return false;
   }
-  if (url.indexOf('signin') === -1) {
+  if (url.indexOf('signin') !== -1) {
```

Comment:
r: url에 `signin`이 포함되어있는지를 찾기위해 [Array.indexOf](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)를 써서 찾아주셨네요. indexOf도 문자열에 특정 부분문자열이 포함되어 있는지를 찾기 위해 사용할 수 있지만, [Array.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)를 사용하면 -1을 비교하지 않아도 true/false값을 바로 알수 있어서 가독성도 좋고 명시적일 것 같아요.

```suggestion
  if (url.includes('signin')) {
```